nailgun/entities.py

```diff
@@ -6849,7 +6857,10 @@ def read(self, entity=None, attrs=None, ignore=None, params=None):
 
     def update_payload(self, fields=None):
         """Wrap submitted data within an extra dict."""
-        return {u'subnet': super(Subnet, self).update_payload(fields)}
+        payload = super(Subnet, self).update_payload(fields)
+        if 'from_' in payload:
+            payload['from'] = payload.pop('from_')
```

Comment:
would this also work here?
```python
payload['from'] = payload.pop('from_', payload['from'])
```

Reply:
Ahh looks like you're replacing from with from_ internally, but updating the payload back to from. What was the reasoning behind this?
Also, that is just to make it a one-liner

Reply:
see the bullet point 1 in the description: https://github.com/SatelliteQE/nailgun/issues/606

Reply:
ahhh gotcha. that is annoying, but unavoidable without a change to the API